sql/hive/src/test/scala/org/apache/spark/sql/hive/DynamicPartitionPruningHiveScanSuite.scala

```diff
@@ -33,12 +32,6 @@ abstract class DynamicPartitionPruningHiveScanSuite
 
   override protected def collectDynamicPruningExpressions(plan: SparkPlan): Seq[Expression] = {
     flatMap(plan) {
-      case s: FileSourceScanExec => s.partitionFilters.collect {
-        case d: DynamicPruningExpression => d.child
-      }
-      case s: BatchScanExec => s.runtimeFilters.collect {
-        case d: DynamicPruningExpression => d.child
-      }
```

Comment:
not related to this change, but better to also rename `DynamicPartitionPruningHiveScanSuite` to `DynamicPartitionPruningHiveScanSuiteBase`